closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -225,6 +231,25 @@ public static boolean isFIPSEnabled() {
         return false;
     }
 
+    /**
+     * Check if the FIPS mode set is 140-2.
+     *
+     * Older profiles are set in FIPS 140-2 mode. This method responds on
+     * whether the loaded profile is one of those.
+     *
+     * @return true if the FIPS mode set is 140-2
+     */
+    public static boolean isFIPS1402() {
+        if (securityEnabled) {
+            return "140-2".equals(restricts.jdkFipsMode);
+        }
+        return false;
+    }
+
+    private static boolean isFIPS1402Profile(String profileID, Properties securityProps) {
+        return "140-2".equals(securityProps.getProperty(profileID + ".fips.mode"));
+    }
```

Comment:
This can be an instance method in `ProfileParser` so the `Properties` object is implied.

Reply:
Moved.